### DISCUSSION TITLE
Welcome to toriis Discussions!

### DISCUSSION THREAD

**Discussion by kennykos** [toriis-portal/toriis#197] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our [toriis.earth](https://www.toriis.earth/home) community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## :link: Links
  Here are some relevant links to help you contribute to this community.

- [Issue Template](https://github.com/toriis-portal/toriis/blob/main/.github/ISSUE_TEMPLATE/developer-issue.md)
- [Commit Message Standards](https://gist.github.com/tonibardina/9290fbc7d605b4f86919426e614fe692)

## :speech_balloon: About Toriis
👋 [toriis.earth ](https://www.toriis.earth/home)🌎 is a website that aims to promote environmental awareness and inspire action for the preservation of our planet by use of user-friendly & interactive data visualizations of institutional investments 📊 through an environmental lens 🌱

toriis.earth was created by [Students for Environmental Concerns](https://secsatuiuc.web.illinois.edu/) (SECS) and [Hack4Impact UIUC](https://uiuc.hack4impact.org/about) (H4I) as a result of a grant from the Illinois Green Fund awarded by the [University of Illinois Student Sustainability Committee](https://www.studentaffairs.illinois.edu/about/committees/student-sustainability-committee), and is currently run by two University of Illinois (UIUC) alumni @kennykos and @faithlos. 

Questions? Here’s how to reach us:

- Email: info@toriis.com
- Toriis Team : @kennykos @faithlos 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kennykos** [comment]
Hi, I'm [Gabriel](https://kennykos.github.io/), one of the members of the toriis team. I am a University of Illinois Alum, and I began working on quantifying the environmental impact of the [University of Illinois Systems Investments](https://www.treasury.uillinois.edu/investments/) in 2021, when I worked with SECS members to publish the [2021 UIS Investment Data Report](https://secsatuiuc.web.illinois.edu/2021/12/01/uis-investment-data-report-2021/). With the help of a team from [Hack 4 Impact UIUC](https://uiuc.hack4impact.org/about), run by @zorazrr and @jocelynxu01, this report involved into the [toriis](https://github.com/toriis-portal/toriis) project, which I am so happy to be able to collaborate on with you all :smile:

**Reply by faithlos** [reply]
Hi! 👋 I'm [Faith](https://faithlosbanes.weebly.com/) :) I am also a recent University of Illinois Urbana-Champaign grad. I have been working on the TORIIS project since the fall of 2022 where I started as a liaison between Students for Environmental Concerns (SECS) and Hack4Impact UIUC. I have been managing aspects of content, design, and development for the website at various levels, and as such, **my background can provide support for the product research/design side of things**. 

So many passionate folks have contributed to the foundations of this project and I am so excited to see our little project grow!  🤗🌱
